airflow/models/dag.py

```diff
@@ -749,7 +749,7 @@ def __init__(
         self.jinja_environment_kwargs = jinja_environment_kwargs
         self.render_template_as_native_obj = render_template_as_native_obj
 
-        self.doc_md = self.get_doc_md(doc_md)
```

Comment:
I think that `self.` is still needed here.

Reply:
Yeah, good catch

Reply:
opps, not the best idea to send a PR before bed 🤦‍♂️ 

Thanks for reminding me! Just fixed it.